src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -40,7 +44,33 @@
      * София - 900</p>
      */
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
-            throws Exception {
-        return null;
+            throws IllegalDateParametersException {
+
+        final DateTimeFormatter DataFormatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
it's not constant 
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Reply:
done